src/OpenTelemetry.SemanticConventions/scripts/generate.ps1

```diff
@@ -0,0 +1,30 @@
+$SCRIPT_DIR = $PSScriptRoot
+$ROOT_DIR = "${SCRIPT_DIR}/../"
+
+# freeze the spec & generator tools versions to make SemanticAttributes generation reproducible
```

Comment:
You have "latest" now for $GENERATOR_VERSION so this comment is lying 🤣 I think either go back to a fixed version for generator or just update the comment.